python/ql/src/Expressions/UseofApply.ql

```diff
@@ -10,8 +10,9 @@
  */
 
 import python
+private import semmle.python.types.Builtins
 
 from CallNode call, ControlFlowNode func
 where
-major_version() = 2 and call.getFunction() = func and func.refersTo(Object::builtin("apply"))
-select call, "Call to the obsolete builtin function 'apply'."
+major_version() = 2 and call.getFunction() = func and func.pointsTo(Value::named("apply"))
+select call, "Call to the obsolete builtin function 'apply'."
```

Comment:
You should run this through the "QLformat for Python" script, so we get the benefit of that lovely, lovely autoformatting. :slightly_smiling_face:

Reply:
Done!